uint16 VehicleSpeed
uint8 VehBatteryStatus
int8 AutoMode
uint8 VehDriveMode
float32 Latitude
float32 Longitude
uint8 BrakeMode
int8 Diagnostic
string JuncAhead
string JuncReached
string StopAhead
string StopReached
uint8 TripStatus
uint8 JunctionStatus
uint8 StopStatus
uint8 SafeToDrive
uint8 ResumeTrip
uint8 JunctionApproaching